lib/watir/attribute_helper.rb

```diff
@@ -38,6 +38,10 @@ def attribute_list
     #  @return [$1] value of $3 property
     #
     def attribute(type, method, attr)
+      if %i(thead tfoot).include? method
```

Comment:
Just tried it and it works correctly.

You probably used `thead` instead of `tHead`. Ignored attributes should be IDL ones.